lib/asset/baseAsset.js

```diff
@@ -61,9 +61,7 @@ class BaseAsset {
 
 
     /**
-     * 20-byte hex ID of the document
-     * @type string
-     * @readonly
+     * @returns {string} - 20-byte hex ID of the document
```

Comment:
Are you sure about this? `asset_id` is a property, so it seems to me that the jsdoc rule should not warn about this.

Reply:
You are right. Eslint was complaining but I will check why.

Reply:
The exact Eslint error was js-doc syntax error. It was because eslint wants `@type {string}`, not `@type string`. I reverted to previous state and added the curly brackets.